Application/Frontend/pages/api/webhooks/clerk-mongodb-sync.ts

```diff
@@ -121,17 +121,18 @@ export default async function POST(req: Request) {
             }
             break;
           case 'user.deleted':
-            // TODO: Delete user from MongoDB
+            // Delete user in MongoDB
             console.log(`Deleting user: ${postData.firstName} ${postData.lastName}`);
             try {
               const response = await fetch('http://localhost:3000/api/delete-user', {
                 method: 'POST',
                 headers: {
                   'Content-Type': 'application/json',
                 },
-                body: JSON.stringify(postData.email),
+                // Stringify the whole object, not just the id
+                body: JSON.stringify({ id: postData.id }), // postData contains the id field
               });
-          
+              
               if (!response.ok) {
```

Comment:
- Enhance error handling to provide more informative error messages and to handle different error scenarios appropriately. For example, in the user.deleted case, ensure that errors from the API call are properly handled, and consider providing more context about the nature of the error.